### PR TITLE
chore: add num_grpc_channels and batch_size settings

### PR DESCRIPTION
**Pull request by pgautier404**
This commit:
1. Adds two new settings to the `settings.php` file that are, for now, otherwise undocumented. They are:
`$settings['momento_cache']['num_grpc_channels']` and `$settings['momento_cache']['batch_size']`, and they default to 1 and 50, respectively.

2. Use latest version of php sdk, and updates the plugin to use getBatch/setBatch APIs from momento SDK instead of looping over get/set apis for making multiple requests.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,6 +8,6 @@
         }
     },
     "require": {
-        "momentohq/client-sdk-php": "1.6.0"
+        "momentohq/client-sdk-php": "v1.7.0"
```
Comment (cprice404) [inline composer.json]: is there a reason why this one has a `v` and the prev one didn't?

**Review by cprice404 — DISMISSED** [review]
is this the version that you all were using in your load testing?

it looks good to me, just want to make sure that we're not shifting the ground from underneath the data we pulled in the load tests. so I'd like either @rishtigupta or @bruuuuuuuce or both to review, but I'm good with it if they are.

File: composer.json (hunk 1 above)
Comment (rishtigupta) [reply composer.json]: No, both are acceptable

File: README.md [hunk 2]
```diff
@@ -4,7 +4,10 @@
 
 A Momento API Token is required. You can generate one using the [Momento Console](https://console.gomomento.com/).
 
-The Momento Cache module uses [Momento's PHP SDK](https://docs.momentohq.com/cache/develop/sdks/php) internally. While installing the Drupal module will automatically install the SDK for you, you will need to install and enable [the PHP gRPC extension](https://github.com/grpc/grpc/blob/master/src/php/README.md) separately for the SDK to function.
+The Momento Cache module uses [Momento's PHP SDK](https://docs.momentohq.com/cache/develop/sdks/php) internally. While installing the Drupal module will automatically install the SDK for you. Separately, you will need to install and enable the following extensions:
```
Comment (bruuuuuuuce) [inline README.md]: The link is not working for me

Comment (rishtigupta) [reply README.md]: Fixed

**Review by bruuuuuuuce — APPROVED** [review]
:shipit:

**Pull request merged**